python/lsst/ip/isr/isrTask.py

```diff
@@ -2541,18 +2543,21 @@ def roughZeroPoint(self, exposure):
             Exposure to process.
         """
         filterLabel = exposure.getFilterLabel()
-        if filterLabel in self.config.fluxMag0T1:
-            fluxMag0 = self.config.fluxMag0T1[filterLabel]
+        physicalFilter = isrFunctions.getPhysicalFilter(filterLabel, self.log)
+
+        if physicalFilter in self.config.fluxMag0T1:
```

Comment:
Using `isrFunctions.checkFilter` could be used here as well.

Reply:
I think I retract this comment, because the physicalFilter is needed multiple times (as this isn't just a simple check).